github/helpers/githubActivityGraphURL.ts

```diff
@@ -1,4 +1,4 @@
 
 export function githubActivityGraphUrl(username : string): string {
-    return `https://activity-graph.herokuapp.com/graph?username=${username}&bg_color=ffffff&color=708090&line=24292e&point=24292e`
+    return `https://github-readme-activity-graph.cyclic.app/graph?username=${username}&bg_color=ffffff&color=708090&line=24292e&point=24292e&area=true&hide_border=true`
 }
```

Comment:
@chinma-yyy Can you please add a comment, linking the project that we are using for this so this remains documented incase they change the URL ?